### PR TITLE
feat: add Arbitrum deployment

### PR DESCRIPTION
**Pull request by iherger**
## PR-Codex overview
The focus of this PR is to add support for the Arbitrum deployment. 

### Detailed summary
- Added Arbitrum deployment to the environment
- Updated network definitions for Arbitrum
- Modified client URL logic for Arbitrum
- Defined assets and releases for Arbitrum
- Adjusted deployment checks for Arbitrum

> The following files were skipped due to too many changes: `packages/environment/src/assets/arbitrum.ts`, `packages/environment/src/deployments/arbitrum.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by ignacio-rsg** [comment]
Pair | Feed Address | Asset Name | Asset Address
-- | -- | -- | --
CBETH / ETH | 0xa668682974E3f121185a3cD94f00322beC674275 | Coinbase Wrapped Staked ETH | 0x1debd73e752beaf79865fd6446b0c970eae7732f
ezETH / ETH | 0x11E1836bFF2ce9d6A5bec9cA79dc998210f3886d | Renzo Restaked ETH | 0x2416092f143378750bb29b79ed961ab195cceea5
SOL / USD | 0x24ceA4b8ce57cdA5058b924B9B9987992450590c | Solana | 0x2bcc6d6cdbbdc0a4071e48bb3b969b06b3330c07
WBTC / USD | 0xd0C7101eACbB49F3deCcCc166d238410D6D46d57 | Wrapped Bitcoin | 0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f
RDNT / USD | 0x20d0Fcab0ECFD078B036b6CAf1FaC69A6453b352 | Radiant Capital | 0x3082cc23568ea640225c2467653db90e9250aaa0
weETH / ETH | 0xE141425bc1594b8039De6390db1cDaf4397EA22b | Wrapped eETH | 0x35751007a407ca6feffe80b3cb397736d2cf4dbe
RSETH / ETH | 0xb0EA543f9F8d4B818550365d13F66Da747e1476A | Kelp DAO Restaked ETH | 0x4186bfc76e2e237523cbc30fd220fe055156b41f
AVAX / USD | 0x8bf61728eeDCE2F32c456454d87B5d6eD6150208 | Avalanche | 0x565609faf65b92f7be02468acf86f8979423e514
WSTETH / ETH | 0xb523AE262D20A936BC152e6023996e46FDC2A95D | Wrapped stETH | 0x5979d7b546e38e414f7e9822514be443a4800529
ARB / USD | 0xb2A824043730FE05F3DA2efaFa1CBbe83fa548D6 | Arbitrum | 0x912ce59144191c1204e64559fe8253a0e49e6548
BNB / USD | 0x6970460aabF80C5BE983C6b74e5D06dEDCA95D4A | BNB | 0xa9004a5421372e1d83fb1f85b0fc986c912f91f3
OP / USD | 0x205aaD468a11fd5D34fA7211bC6Bad5b3deB9b98 | Optimism | 0xac800fd6159c2a2cb8fc31ef74621eb430287a5a
USDC / USD | 0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3 | Circle USD | 0xaf88d065e77c8cc2239327c5edb3a432268e5831
AAVE / USD | 0xaD1d5344AaDE45F43E596773Bcc4c423EAbdD034 | Aave | 0xba5ddd1f9d7f570dc94a51479a000e3bce967196
SWETH / ETH | 0x05Bc6e5Fb110589bb366A3Cd7CdBe143EeBA2168 | Swell Ethereum | 0xbc011a12da28e8f0f528d9ee5e7039e22f91cf18
DAI / USD | 0xc5C8E77B397E531B8EC06BFb0048328B30E9eCfB | DAI | 0xda10009cbd5d07dd0cecc66161fc93d7c9000da1
RETH / ETH | 0xD6aB2298946840262FcC278fF31516D39fF611eF | Rocket Pool ETH | 0xec70dcb4a1efa46b8f2d97c310c9c4790ba5ffa8
LINK / ETH | 0xb7c8Fb1dB45007F98A68Da0588e1AA524C317f27 | Chainlink | 0xf97f4df75117a78c1a5a0dbb814af92458539fb4
UNI / USD | 0x9C917083fDb403ab5ADbEC26Ee294f6EcAda2720 | Uniswap | 0xfa7f8980b0f1e64a2062791cc3b0871572f1f7f0
GMX / USD | 0xDB98056FecFff59D032aB628337A4887110df3dB | GMX | 0xfc5a1a6eb076a2c7ad06ed22c90d7e710e35ad0a
USDT / USD | 0x3f3f5dF88dC9F13eac63DF89EC16ef6e7E25DdE7 | USDT | 0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9

File: packages/environment/package.json [hunk 1]
```diff
@@ -28,8 +28,10 @@
   },
   "devDependencies": {
     "@enzymefinance/sdk": "workspace:*",
+    "dotenv-cli": "^7.4.2",
```
Comment (KedziaPawel) [inline packages/environment/package.json:31]: why do we add those two libs? I don't see usage

File: packages/environment/src/releases.ts [hunk 2]
```diff
@@ -120,13 +128,44 @@ export interface DeploymentNamedTokensPolygon {
...
+  : TDeployment extends Deployment.ETHEREUM
+    ? DeploymentNamedTokensEthereum
+    : TDeployment extends Deployment.POLYGON
       ? DeploymentNamedTokensPolygon
-      : never;
+      : TDeployment extends Deployment.TESTNET
+        ? DeploymentNamedTokensPolygon
+        : never;
+
+export interface DeploymentNamedTokensAssetsArbitrum {
```
Comment (KedziaPawel) [inline packages/environment/src/releases.ts:141]: named tokens are mln, and weth only. Strange that typescript is not complaining

File: packages/environment/package.json (hunk 1 above)
Comment (iherger) [reply packages/environment/package.json]: Removed.

File: packages/environment/src/releases.ts (hunk 2 above)
Comment (iherger) [reply packages/environment/src/releases.ts:141]: Updated.

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**